### PR TITLE
Fix validation regex

### LINKED ISSUE #125: Alpha bug: Email domain for owner
Based on chatgpt, an email domain must include at least one dot (.) and cannot start or end with a dot, should this requirement be reflected in PawPatrol? (maybe we can catch this in other grp's tp during pe)

Screenshot 2024-11-03 at 9 54 33 PM

### LINKED ISSUE #147: Alpha bug: Incorrect owner usage message
```
Invalid command format! 
owner: Adds an owner to PawPatrol. Parameters: i/IC_NUMBER n/NAME p/PHONE e/EMAIL a/ADDRESS Example: owner n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 
```

Example does not include IC_NUMBER

### LINKED ISSUE #148: Alpha bug: Some names not supported
Owner does not support actual names such as Indian names (e.g. names containing "s/o") and Arabic names

### LINKED ISSUE #160: Alpha bug: Example command for add owner may be missed at first glance
The user will need to scroll to the right to view the example command for adding an owner. The example could be placed on a new line to improve readability.

### PR DESCRIPTION
**Pull request by jloh02**
Updates to allow multilingual texts and closes #148  and fixes #147 and fixes #160 and fixes #125

### CODE REVIEW

File: src/main/java/seedu/address/model/owner/Address.java [hunk 1]
```diff
@@ -9,13 +9,13 @@
  */
 public class Address {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Addresses must be of length 3";
```
Comment (clarissatjx) [inline src/main/java/seedu/address/model/owner/Address.java:12]: clearer to say "Address must be at least 3 characters long" something like that?

**Review by clarissatjx — APPROVED** [review]
other than the comment i left, lgtm. we need update user guide also (to indicate email must be of certain format and phone no must be 8 digits long and address must be min 3 char long)

File: src/main/java/seedu/address/model/owner/Email.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ public class Email {
     private static final String DOMAIN_PART_REGEX = ALPHANUMERIC_NO_UNDERSCORE
         + "(-" + ALPHANUMERIC_NO_UNDERSCORE + ")*";
     private static final String DOMAIN_LAST_PART_REGEX = "(" + DOMAIN_PART_REGEX + "){2,}$"; // At least two chars
-    private static final String DOMAIN_REGEX = "(" + DOMAIN_PART_REGEX + "\\.)*" + DOMAIN_LAST_PART_REGEX;
+    private static final String DOMAIN_REGEX = "(" + DOMAIN_PART_REGEX + "\\.)+" + DOMAIN_LAST_PART_REGEX;
```
Comment (Runus01) [inline src/main/java/seedu/address/model/owner/Email.java:31]: Good idea to edit this to ensure that all emails keyed in are valid

**Review by Runus01 — APPROVED** [review]
LGTM! Thanks for updating all these regex!

**Review by naythee169 — APPROVED** [review]
LGTM

**Pull request merged**